engine/src/main/java/org/destinationsol/FactionDisplay.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol;
+
+
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.math.Vector3;
+import org.destinationsol.common.SolMath;
+import org.destinationsol.game.FactionInfo;
+import org.destinationsol.game.SolCam;
+import org.destinationsol.game.SolGame;
+import org.destinationsol.game.screens.MapScreen;
+import org.destinationsol.game.ship.SolShip;
+import org.destinationsol.ui.DisplayDimensions;
+import org.destinationsol.ui.UiDrawer;
+
+
+public class FactionDisplay {
+    private SolCam cam;
+    private DisplayDimensions dimensions;
+    private FactionInfo info;
+    private SolShip solShip;
+    private MapScreen mapScreen;
+
+    private boolean isPressed = false;
+
+    public FactionDisplay(SolGame game, FactionInfo factionInfo) {
+        dimensions = SolApplication.displayDimensions;
+        cam = game.getCam();
+        info = factionInfo;
+    }
+
+    public void drawFactionNames(UiDrawer uiDrawer){
+        if (Gdx.input.isKeyJustPressed(Input.Keys.Z))
```

Comment:
We follow 1TBS exclusively https://github.com/MovingBlocks/Terasology/wiki/Code-Conventions

Reply:
(You missed braces)